Signal/src/Models/TSMessageAdapaters/TSMessageAdapter.m

```diff
@@ -217,7 +217,8 @@ - (NSString *)senderId {
 }
 
 - (NSString *)senderDisplayName {
-    if (self.thread) {
+    BOOL isSenderMe = [_senderDisplayName isEqualToString:NSLocalizedString(@"ME_STRING", @"")];
```

Comment:
I think `[self.interaction isKindOfClass:[TSOutgoingMessage class]]` would be less brittle